docs/datamodel/links.rst

```diff
@@ -143,7 +144,7 @@ instances can link to the same target(s).
 
 In the ``GroupChat`` example, the ``GroupChat.members`` link is now
 ``exclusive``. Two ``GroupChat`` objects cannot link to the same ``Person``;
-put differently, no ``Person`` can be a ``member`` of multiple ``GroupChat``.
+put differently, no ``Person`` can be a ``member`` of multiple ``GroupChat``s.
```

Comment:
I wasn’t sure the best way to make `GroupChat` plural. Could also be changed to "`GroupChat` objects."

Reply:
I think I remember @raddevon mentioning that s for plural right after double backticks often upsets the formatter. So ``GroupChat`` objects is probably the safer option.

Reply:
Sounds good. I've updated it to "`GroupChat` objects".